### ISSUE TITLE
Seems like certificate has been revoked

### ISSUE THREAD

**Issue by smtrd3** [perki/backloop.dev#5]
I am getting ERR_CERT_REVOKED error, the certificate might be revoked as per the error message.

![Screenshot 2024-11-13 at 3 14 46 PM](https://github.com/user-attachments/assets/ee54429c-d450-4b29-a063-f6649684167a)

**Comment by perki** [comment]
Indeed the following command `openssl ocsp -issuer certs/backloop.dev-ca.crt -cert certs/backloop.dev-cert.crt -text -url http://r10.o.lencr.org` returns `keyCompromise`

This is probably due to the fact that the key is publicly available .. Which is the purpose of backloop.dev 

I need to figure out how it was reported and how to avoid this

**Comment by perki** [comment]
@smtrd3  I renewed the certificates. 
You just have to delete the curent ones (in `certs/*`) and run `npm run postinstall` to install the new ones, still this is a quick fix and we can expect the certificates to be revoked until we find the cause and a remedy ;)

**Comment by perki** [comment]
@smtrd3  Actually, anyone can revoke the certificate. As the key is available [see](https://letsencrypt.org/docs/revoking/#using-the-certificate-private-key) 

This might be of a good intent .. but not in the case of `backloop.dev`

**Comment by zeroskylian** [comment]
<img width="790" height="188" alt="Image" src="https://github.com/user-attachments/assets/5cc0b8a8-ef92-4aaf-99f3-60d3c2fa3265" />

Hello, has the certificate been revoked?

**Comment by perki** [comment]
@zeroskylian  Yes, this might happend. Saddly anyone with the cert can revoke it manually. 
New certificates are issued every two days to cope with this.
This is why you have now the command `backloop.dev-update` if backloop is installed in global.
If installed as a package `rm node_modules/backloop.dev/certs/backloop.dev*` will erase current certificate and force reload.

**Comment by zeroskylian** [comment]
> [@zeroskylian](https://github.com/zeroskylian) Yes, this might happend. Saddly anyone with the cert can revoke it manually. New certificates are issued every two days to cope with this. This is why you have now the command `backloop.dev-update` if backloop is installed in global. If installed as a package `rm node_modules/backloop.dev/certs/backloop.dev*` will erase current certificate and force reload.

OK, I understand. 
Next action will run, I'll wait for a while.

**Comment by zeroskylian** [comment]
<img width="782" height="178" alt="Image" src="https://github.com/user-attachments/assets/c49d8edd-8df8-4498-8045-06d176a8fa03" />
